sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -261,6 +261,7 @@ static bool isDeviceBinaryTypeSupported(const context &C,
   if (Format != PI_DEVICE_BINARY_TYPE_SPIRV)
     return true;
 
+  pi::Backend CBackend = (detail::getSyclObjImpl(C)->getPlugin()).getBackend();
```

Comment:
Could you please move this liner closer to where the var is used?

Reply:
Fixed.